D4Group.cc

```diff
@@ -162,6 +162,22 @@ D4Group *D4Group::find_child_grp(const string &grp_name) {
     return (g == grp_end()) ? 0 : *g;
```

Comment:
While we're in this code, the '0' here should be nullptr.

Reply:
Yes, correct it.